test/start-stop.make

```diff
@@ -40,8 +40,9 @@ start: _work/clear-kvm.img _work/kube-clear-kvm _work/start-clear-kvm _work/ssh-
 		touch _work/clear-kvm.secretsdone; \
 	fi
 	_work/ssh-clear-kvm kubectl version --short | grep 'Server Version' | sed -e 's/.*: v\([0-9]*\)\.\([0-9]*\)\..*/\1.\2/' >_work/clear-kvm-kubernetes.version
+	. test/test-config.sh && \
```

Comment:
This line was added as one last change just now, @pohly you agree it's ok? (it somehow had escaped my earlier testing, but we need to get TEST_DEVICEMODE to env here as well)